cypress/integration/main_page_spec.js

```diff
@@ -1,27 +1,71 @@
+const getAllMovieData = () => {
+  return {
+    statusCode: 200,
+      ok: true,
+      body: {
+        movies: [{
+          id: 694919,
+          poster_path: "https://image.tmdb.org/t/p/original//6CoRTJTmijhBLJTUNoVSUNxZMEI.jpg",
+          backdrop_path: "https://image.tmdb.org/t/p/original//pq0JSpwyT2URytdFG0euztQPAyR.jpg",
+          title: "Money Plane",
+          average_rating: 6.625,
+          release_date: "2020-09-29"
+        }]
+      }
+  }
+}
+
```

Comment:
I'm still so proud of us for pulling out this reused code into it's own function